platformio/src/main.cpp

```diff
@@ -186,8 +186,27 @@ void setup()
   tm timeInfo = {};
 
   // START WIFI
+  wl_status_t wifiStatus;
   int wifiRSSI = 0; // “Received Signal Strength Indicator"
-  wl_status_t wifiStatus = startWiFi(wifiRSSI);
+  
+  pinMode(PIN_CONFIGURE_WIFI, INPUT_PULLUP);
+  if (digitalRead(PIN_CONFIGURE_WIFI) == LOW) 
```

Comment:
Inspecting this pins status to either try to connect to the network normally or go into config AP mode.